mailpoet/assets/js/src/common/set_from_address_modal.tsx

```diff
@@ -143,7 +143,7 @@ function SetFromAddressModal({ onRequestClose, setAuthorizedAddress }: Props) {
             removeUnauthorizedEmailNotices();
             notices.success(getSuccessMessage(), { timeout: false });
           } catch (e) {
-            const error = e.errors && e.errors[0] ? e.errors[0] : null;
+            const error:error|null = e.errors && e.errors[0] ? e.errors[0] : null;
```

Comment:
OK. Yep, `e` is `unknown` because at runtime (in the JS) there is no way to check for the type or catch only some types. There indeed should be a runtime check for the type you expect but `error` is TS-only type (not a runtime class) so this can't be done easily here.